### DISCUSSION TITLE
Migrating selective pages over to sharepoint

### DISCUSSION THREAD

**Discussion by L3m0nRoCk** [WikiTransformationProject/wikitraccs-releases#124] [Q&A] [answered]
Hi everyone.
Is it possible to, instead of migrating an entire confluence space over to sharepoint, select pages from confluence to be migrates. lets say I only want to migrate a subsection of pages from a confluence page over to sharepoint? does such a feature allready exist?

**Comment by heinrich-ulbricht** [comment]
Hi @L3m0nRoCk,

how would you like to select those pages? By a list of page IDs? Or some other dynamic criteria, like modification date? Such a feature doesn't exist at the moment.

For partial migrations clients choose among the following options:
* migrate the whole space, then delete pages in SharePoint that are not needed
* use CQL selectors to migrate selected pages (but beware of page restrictions); see [How to migrate Confluence Pages using CQL Query Selectors](https://www.wikitransformationproject.com/docs/reference/settings/confluence-space-inventory/confluence-cql-query-selectors/) for details

Would those work for you?

**Reply by L3m0nRoCk** [reply]
Hi thanks for the quick reply.
A list of page IDs would be the easiets to implement but for those not so computer savy would find getting page ID's a bit hard.

Probebly a list of page names that the user can select would be best but does make the list not all unique values. That could pose a problem.

Another solution to the problem is to create a seperate space in confluence with the required pages. Updating the wiki tracks sharepoint and then migrating that temp space.

But let me read into the CQL.

**Comment by heinrich-ulbricht** [comment, marked as answer]
Note: The [Content ID Selector](https://www.wikitransformationproject.com/docs/reference/settings/confluence-space-inventory/confluence-content-id-selectors/) now allows to select content IDs (page, blogpost, ...) to migrate.
